src/msg.h

```diff
@@ -96,7 +97,7 @@ typedef struct {
 	uint8_t  type;     /**< Must be \ref BL_MSG_SAMPLE_DATA */
 	uint8_t  channel;  /**< Channel of sample data. */
 	uint8_t  count;    /**< Number of samples in packet. */
-	uint8_t  reserved;
+	uint8_t  src_led;  /**< Source LED of sample data. */
```

Comment:
I'd probably put this above count, because it makes sense for count to come before the sample data.

Reply:
+1